File: intro/python/2_century_from_year/century_from_year.py
"""Solution for centuryFromYear."""


def century_from_year(year: int) -> int:
    """Figure out which century a given year would be in."""
    if year % 100 == 0:
        return year // 100
    else:
        return (year // 100) + 1


year = 2405
print(f"{year} is in the {century_from_year(year)}th century.")

File: intro/go/2_centuryFromYear/centuryFromYear.go
package main

import "fmt"

func centuryFromYear(year int) int {
	if year%100 == 0 {
		return year / 100
	}
	return (year / 100) + 1
}

func main() {
	fmt.Printf("The year 2045 is in the %dt century.\n", centuryFromYear(2045))
}
